assets/api-swagger/swagger.json

```diff
@@ -23,23 +23,29 @@
           "content": {
             "application/json": {
               "schema": {
-                "oneOf": [
-                  {
-                    "$ref": "#/components/schemas/Contact"
-                  },
-                  {
-                    "$ref": "#/components/schemas/CustomEvent"
-                  },
-                  {
-                    "$ref": "#/components/schemas/Order"
-                  },
-                  {
-                    "$ref": "#/components/schemas/Product"
-                  },
-                  {
-                    "$ref": "#/components/schemas/Profile"
-                  }
-                ]
+                "title": "Push Objects",
+                "type": "array",
+                "items": {
+                  "type": "object",
+                  "description": "Send 100 records of either Contact, Profile to Ometria",
```

Comment:
This isn't just Contract and Profile

Reply:
yes that's just a trial thing,  want to see if the description is visible anywhere so Louise can then insert some copy here.